climpred/prediction.py

```diff
@@ -38,19 +47,62 @@ def compute_perfect_model(
         metric (str): `metric` name, see :py:func:`climpred.utils.get_metric_function`.
         comparison (str): `comparison` name, see
                           :py:func:`climpred.utils.get_comparison_function`.
+        dim (str or list): dimension to apply metric over. default: ['member', 'init']
         add_attrs (bool): write climpred compute args to attrs. default: True
 
```

Comment:
Add entry for `**kwargs` here maybe? Maybe `**kwargs` should be renamed to `**metric_kwargs` to be more explicit.